pkg/cmd/roachtest/tests/tpcc.go

```diff
@@ -740,6 +740,24 @@ func registerTPCC(r registry.Registry) {
 		},
 	})
 
+	r.Add(registry.TestSpec{
+		Name:                      "tpcc-nowait/w=1000/nodes=5/cpu=16",
+		Owner:                     registry.OwnerTestEng,
+		Benchmark:                 true,
+		Cluster:                   r.MakeClusterSpec(6, spec.CPU(16), spec.WorkloadNode()),
```

Comment:
just a heads up that the workload node will be 4vcpu. You could add a WorkloadNodeCPUs(16 or 8) option here.
I have no idea if 4 vcpus is too few. Just pointing it out, since I found it surprising to read about the 4vcpu default in the comments for WorkloadNode.

Reply:
Thanks for this tip.  I ran some additional tests and found that the 4 vCPU workload node is sufficient (we have 50% headroom) but I also noticed a couple of other problems with the test I'd written:

1. We were overloading the CRDB cluster with workers, so I dialed the number of workers down to 200 (from the default of `number of warehouses`).  That juiced performance from ~70K to ~100K.
2. There was a backup schedule being created for the run, which was undesirable.
3. I also added `--tolerate-errors`, as we're not using this for correctness testing but rather, for benchmarking only.

I've made those changes and am hoping you can have a quick third look.